src/tribol/interface/tribol.cpp

```diff
@@ -462,6 +580,18 @@ void registerNodalVelocities( integer meshId,
 
 } // end registerNodalVelocities()
 
+//------------------------------------------------------------------------------
+void registerMfemVelocity( integer cs_id, const mfem::ParGridFunction &v )
+{
+   auto coupling_scheme = CouplingSchemeManager::getInstance().getCoupling(cs_id);
+   SLIC_ERROR_ROOT_IF(
+      !coupling_scheme->hasMfemData(), 
+      "Coupling scheme does not contain MFEM data. "
+      "Was the coupling scheme created with registerParMesh()?"
+   );
+   coupling_scheme->getMfemMeshData()->SetParentVelocity(v);
```

Comment:
Up to this point I haven't seen "Parent" used. Let's choose one and stick with it. We have ParMesh, SubMesh, LORMesh, and RedecompMesh. If we want to call ParMesh the ParentMesh let's stay consistent.

Reply:
Based on the hierarchy introduced, this will be referred to as parent.